sql/00-create-user.sql

```diff
@@ -1,7 +0,0 @@
-CREATE extension ltree;
-CREATE DATABASE blobber_meta;
-
-\connect blobber_meta;
-CREATE USER blobber_user WITH ENCRYPTED PASSWORD 'blobber';
```

Comment:
Does the deploy team know the password 'blobber' used here, if not please notify them to remember change the password when deploy to production env.

Reply:
I've put these values in `0chain_blobber.yaml` file. I will let them know.